spec/commands/new_spec.rb

```diff
@@ -58,13 +58,51 @@ def headers
     expect(output).to eq(expected_output)
   end
 
+  it 'treats loader name to be downcased' do
+    ENV['EDITOR'] = 'emacs'
+
+    allow(File).to receive(:exist?).with('loaders/testing2.rb').and_return(true)
+    allow_any_instance_of(Commands::New).to receive(:system).with('emacs loaders/testing2.rb')
+    expect(File).to_not receive(:open)
+    expect(File).to_not receive(:write)
+
+    expected_output = "Editing loader\n"
+    output = capture_stdout_from { attempt_command('new Testing2') }
+
+    expect(output).to eq(expected_output)
+  end
```

Comment:
This test doesn't look :rock:  solid to me. It breaks whenever I add a `binding.pry`. Can we make these mocks a little :pinching_hand: bit more reliable? :rocket: :sparkles:

Reply:
100% :rock: solid now
Thanks for pointing!